### ISSUE TITLE
Web server black screen when render jobs are present

### ISSUE THREAD

**Issue by VL4DST3R** [webbukkit/dynmap#3434]
> template is **bold**
> sample data is *italicized*

**Issue Description:** *Ever since the existence of issues like #3427 or the prospect of a new map on our multiverse server, this issue has always come in tow when dealing with massive or even multiple world render tasks.*

*While i haven't been able to pinpoint it exactly, seems to be directly correlated with long/large render tasks, be it interrupted or not. This issue causes broken "ghost" tasks: tasks that get saved at the time of closing the server, regardless if they are actually finished or not. Those show up again every time the server restarts and either resume, or, if they are actually done, they complete instantly with just one mention in the server log. But never get removed! so the next server restart it shows up again.*

*When one (or more) of those render tasks present, the web interface stops working altogether, all dimensions are black and no user skin shows up. If i delete those tasks manually, dynmap initializes properly and the web map works, presumably with just the render data still present in those files being lost.*

* **Dynmap Version:** *Dynmap version: core=3.2-beta-2-483, plugin=3.2-beta-2-483*
* **Server Version:** *Paper version git-Paper-642 (MC: 1.16.5) (Implementing API version 1.16.5-R0.1-SNAPSHOT)
* **Pastebin of Configuration.txt:** *https://pastebin.com/Jds2H9rP*
* **Server Host (if applicable):** *self-hosted*
* **Pastebin of crashlogs or other relevant logs:** *n/a*
* **Other Relevant Data/Screenshots:** *This happens regardless of map version and while using no texture pack*
* **Steps to Replicate:** *as stated above, i don't have a guaranteed procedure, but doing a `fullrender` of one or more large dimensions, for many hours, almost always results in this issue sooner or later.*

 [x] *I have looked at all other issues and this is not a duplicate*  
 [x] *I have been able to replicate this **(somewhat)***

**Comment by JurgenKuyper** [comment]
@VL4DST3R is this still an issue in the latest versions?

**Comment by VL4DST3R** [comment]
Yes, this is still an issue if using database storage. Recently I tested by switching to loose files to compare and contrast, and while the crashing/lockup reported here no longer happens, it still seems to generate the task files that never disappear but it does not prevent the rest of the map from working.

Do note this is a separate issue from the other very frequent (and arguably worse for functionality) issue #4001
